gems/pending/openstack/openstack_handle/pagination/marker.rb

```diff
@@ -0,0 +1,25 @@
+require 'openstack_handle/pagination/base'
+
+module OpenstackHandle
+  module Pagination
+    class Marker < OpenstackHandle::Pagination::Base
+      def list
+        all_servers = call_list_method(@collection_type, @options, @method)
```

Comment:
Is this `list` method always specific to `servers`?  If not, change this to something more generic like `all_objects`, maybe.  Otherwise, if this *is* specific to `servers`, then I don't see a reason for it to live in a generic looking `Pagination` module when it should just live in the area where `server` retrieval is defined.